apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/HashingAlgorithms.kt

```diff
@@ -0,0 +1,40 @@
+package com.apollographql.apollo.compiler
+
+import java.math.BigInteger
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.security.MessageDigest
+import java.util.Locale
+
+class HashingAlgorithms(
+    private val algorithm: String?
```

Comment:
why can't this default to SHA256 instead of being nullable?

Reply:
Fixed.